.github/ISSUE_TEMPLATE/organization-membership-request.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Organization Membership Request
+about: Request membership in an Istio Org
+
+---
+
+### GitHub Username
```

Comment:
Isn't this implicit based on who opens the issue?

Reply:
it is, if it is opened by the same person, but we don't require to be the same person.